rosstatic/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.0)
+project(rosstatic)
+
+find_package(catkin REQUIRED)
+
+catkin_package()
+
+install(DIRECTORY launch DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Comment:
You also probably want to install your script (use `catkin_install_python()`: http://docs.ros.org/jade/api/catkin/html/howto/format2/installing_python.html - this will ensure Python 2/3 compatibility across ROS distributions, see http://wiki.ros.org/UsingPython3/SourceCodeChanges)